snomed/com.b2international.snowowl.validation.snomed/src/test/java/com/b2international/snowowl/validation/snomed/GenericValidationRuleTest.java

```diff
@@ -59,6 +60,24 @@
 @RunWith(Parameterized.class)
 public class GenericValidationRuleTest extends BaseGenericValidationRuleTest {
 	
+	@Test
+	public void codeSystemURITest() throws Exception {
```

Comment:
This should be added to the core tests, it should not part of the SNOMED CT Validation Rule module.